docs/network-drivers.md

```diff
@@ -213,6 +215,50 @@ The body of the api call request will look like:
 The brocade driver accepts interface names the same way they would be accepted
 in the console of the switch, ex. ``101/0/10``.
 
+### Juniper QFX5200 driver
+
+#### Switch preparation
+
+1. The driver relies on netconf protocol to manage the switch. 
+It is important switch is configured for responding to netconf connections. 
+With administration privileges, running the following command in 
+configuration mode should set it up for netconf based connections. 
+
+```
+set system services netconf ssh
+```
+
+2. Driver relies on the PyEZ library, a python wrapper that for netconf 
+using `ncc client`. Make sure it is installed before using the driver. 
+
+3. VLANS that HIL will manage should be preconfigured on switch. 
```

Comment:
Do you want to add a bit on how to do that like we do it for all switches?

Reply:
example added in next commit. done